### ISSUE TITLE
Dependency check for xz instead of xz-utils on Arch-based distros

### ISSUE THREAD

**Issue by yochananmarqos** [teejee2008/aptik#4]
When I install aptik on Manjaro Linux, I receive this message:

```
Following packages could not be installed. Please install these manually:
 > xz-utils
```

`xz-utils` no longer exists in the Arch repos as `xz` replaces it.

**Comment by teejee2008** [comment]
will be fixed in next update

**Comment by AntiNSA** [comment]
Where can I find xzutils? I really need to use aptik in Manjaro... Thanks

**Comment by yochananmarqos** [comment]
@AntiNSA As I already mentioned, xz has replaced xz-utils.

**Comment by AntiNSA** [comment]
@yochananmarqos Did you find a way to get it to work? I have 7 students computers I need to update .. and aptik is my main tool... changing all from ubuntu to manjaro since 18.04 mandb has huge problems on ubuntu.....

I thought aptik worked on MAnjaro... finally got Manjaro all tweaked out and ready to copy for the other students.. and when I go to install aptik i find this error : (

**Comment by yochananmarqos** [comment]
@AntiNSA If you already have `xz` installed, ignore the error. You need both `aptik` and `aptik-gtk` installed if you want to use the GUI.

**Comment by AntiNSA** [comment]
can you give me a link to download the gtk *.sh  version? I am new to manjaro..... thanks

**Comment by yochananmarqos** [comment]
@AntiNSA Use the .run file. Same for `aptik-gtk`: https://github.com/teejee2008/aptik-gtk/releases

**Comment by AntiNSA** [comment]
@yochananmarqos  sorry for my ignorance... I changed persmissions to allow to run as program, and then sudo ./aptik-v18.5.1-amd64.run  and I have no aptik-gtk , I only have aptik , the command line version...

can you tell me how to get the aptik-gtk to install in manjaro?

Thanks

**Comment by AntiNSA** [comment]
@yochananmarqos my mistake : )
Thanks for your help. I see its the right link.

**Comment by teejee2008** [comment]
Fixed in v18.5.2
